audit/org_kubernetes.io/roles/prow.viewer.json

```diff
@@ -0,0 +1,323 @@
+    "container.pods.getStatus",
+    "container.pods.list",
+    "container.replicaSets.get",
+    "container.replicaSets.getScale",
+    "container.replicaSets.getStatus",
+    "container.replicaSets.list",
+    "container.replicationControllers.get",
+    "container.replicationControllers.getScale",
+    "container.replicationControllers.getStatus",
+    "container.replicationControllers.list",
+    "container.resourceQuotas.get",
+    "container.resourceQuotas.getStatus",
+    "container.resourceQuotas.list",
+    "container.roleBindings.get",
+    "container.roleBindings.list",
+    "container.roles.get",
+    "container.roles.list",
+    "container.runtimeClasses.get",
+    "container.runtimeClasses.list",
+    "container.scheduledJobs.get",
+    "container.scheduledJobs.list",
+    "container.serviceAccounts.get",
+    "container.serviceAccounts.list",
+    "container.services.get",
+    "container.services.getStatus",
+    "container.services.list",
+    "container.statefulSets.get",
+    "container.statefulSets.getStatus",
+    "container.statefulSets.list",
+    "container.storageClasses.get",
+    "container.storageClasses.list",
+    "container.thirdPartyObjects.get",
+    "container.thirdPartyObjects.list",
+    "container.thirdPartyResources.get",
+    "container.thirdPartyResources.list",
+    "container.tokenReviews.create",
+    "logging.buckets.get",
+    "logging.buckets.list",
+    "logging.exclusions.get",
+    "logging.exclusions.list",
+    "logging.logEntries.list",
+    "logging.logMetrics.get",
+    "logging.logMetrics.list",
+    "logging.logServiceIndexes.list",
+    "logging.logServices.list",
+    "logging.logs.list",
+    "logging.sinks.get",
+    "logging.sinks.list",
+    "logging.usage.get",
+    "monitoring.alertPolicies.get",
+    "monitoring.alertPolicies.list",
+    "monitoring.dashboards.get",
+    "monitoring.dashboards.list",
+    "monitoring.groups.get",
+    "monitoring.groups.list",
+    "monitoring.metricDescriptors.get",
+    "monitoring.metricDescriptors.list",
+    "monitoring.monitoredResourceDescriptors.get",
+    "monitoring.monitoredResourceDescriptors.list",
+    "monitoring.notificationChannelDescriptors.get",
+    "monitoring.notificationChannelDescriptors.list",
+    "monitoring.notificationChannels.get",
+    "monitoring.notificationChannels.list",
+    "monitoring.publicWidgets.get",
+    "monitoring.publicWidgets.list",
+    "monitoring.services.get",
+    "monitoring.services.list",
+    "monitoring.slos.get",
+    "monitoring.slos.list",
+    "monitoring.timeSeries.list",
+    "monitoring.uptimeCheckConfigs.get",
+    "monitoring.uptimeCheckConfigs.list",
+    "resourcemanager.projects.get",
+    "resourcemanager.projects.list",
+    "serviceusage.quotas.get",
+    "serviceusage.services.get",
+    "serviceusage.services.list",
+    "stackdriver.projects.get"
+  ],
+  "name": "organizations/758905017065/roles/prow.viewer",
+  "stage": "ALPHA",
+  "title": "Prow Viewer"
```

Comment:
Looks like a new monitoring roll... I think I saw this in yaml form:
https://github.com/kubernetes/k8s.io/blob/main/infra/gcp/roles/prow.viewer.yaml#L1

Though I'm unsure the tool used to load it into GCP.